pkg/normalize/validate.go

```diff
@@ -321,7 +321,7 @@ func NormalizeAndValidateConfig(config *models.DNSConfig) (errs []error) {
 			}
 
 			// Populate FQDN:
-			//rec.NameFQDN = dnsutil.AddOrigin(rec.GetLabel(), domain.Name)
+			rec.NameFQDN = dnsutil.AddOrigin(rec.GetLabel(), domain.Name)
```

Comment:
Please use rec.SetLabel() as it will set both .Name and .NameFQDN at the same time.